lib/ui_kit/drawer_menu.dart

```diff
@@ -50,3 +55,35 @@ class _DrawerMenuState extends State<DrawerMenu> {
     );
   }
 }
+
+class LogOutButton extends StatelessWidget {
+  final StoreInteraction preference;
```

Comment:
all work with store you should do in repository
and to get some method from repo, you can by using AuthService